CircleCI/CircleCI.json

```diff
@@ -0,0 +1,317 @@
+                },
+                {
+                    "id": null,
+                    "key": "panel635D794D9F73E847",
+                    "title": "Daily Performance",
+                    "visualSettings": "{\"series\":{},\"overrides\":[{\"series\":[\"failed\"],\"queries\":[],\"properties\":{\"color\":\"#bf2121\"}},{\"series\":[\"success\"],\"queries\":[],\"properties\":{\"color\":\"#98eca9\"}}],\"general\":{\"type\":\"line\",\"displayType\":\"smooth\",\"markerSize\":5,\"lineDashType\":\"solid\",\"markerType\":\"none\",\"lineThickness\":2,\"mode\":\"timeSeries\"},\"color\":{\"family\":\"Categorical Default\"}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "$$logsrc \n| json field=_raw \"job.status\" as status\n| json field=_raw \"project.name\" as projectName| where projectName matches \"{{project}}\"\n| withtime status\n| timeslice 1d\n| count by _timeslice, status\n| transpose row _timeslice column status",
+                            "queryType": "Logs",
+                            "queryKey": "A",
+                            "metricsQueryMode": null,
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "parseMode": "Auto",
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": {
+                        "type": "BeginBoundedTimeRange",
+                        "from": {
+                            "type": "RelativeTimeRangeBoundary",
+                            "relativeTime": "-2w"
+                        },
+                        "to": null
+                    },
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                },
+                {
+                    "id": null,
+                    "key": "panelFC518250B2C32947",
+                    "title": "5 Most Recent Failed Workflows",
+                    "visualSettings": "{\"general\":{\"mode\":\"distribution\",\"type\":\"table\"},\"series\":{}}",
+                    "keepVisualSettingsConsistentWithParent": true,
+                    "panelType": "SumoSearchPanel",
+                    "queries": [
+                        {
+                            "queryString": "$$logsrc  \n| json \"project.name\" as project | where project matches \"{{project}}\"\n| json \"workflow.name\" as workflow\n| json \"workflow.status\" as status\n| json \"happened_at\" as at\n| parseDate(at, \"yyyy-MM-dd'T'HH:mm:ss.SSSXXX\") as time_milli\n| formatDate(time_milli, \"MMM dd,  hh:mm a\") as time\n| where status=\"failed\"\n| \"\u274c\" as status\n| withtime time\n| most_recent(time_withtime) as when by project, workflow, status\n| limit 5",
```

Comment:
instead of having | json "field1.name" as field1 | json "field2.name" as field2  in separate lines would be good to consolidate into one | json "field1.name", "field2.name" as field1, field2